prompt.py

```diff
@@ -933,6 +941,31 @@ def run(self):
                 traceback.print_exc()
 
 
+def check_privatenet():
+    """ Check if privatenet is running, and if container is same as the chain file """
+    rpc_settings.setup(settings.RPC_LIST)
+    client = RPCClient()
+    version = client.get_version()
+    print("Privatenet useragent '%s', nonce: %s" % (version["useragent"], version["nonce"]))
+    if not version:
+        logger.error("Error: private network doesn't seem to be running")
+        return
+
+    # Now check if nonce is the same as in the chain path
+    nonce_container = str(version["nonce"])
+    neopy_chain_meta_filename = os.path.join(settings.LEVELDB_PATH, ".privnet-nonce")
+    if os.path.isfile(neopy_chain_meta_filename):
+        nonce_chain = open(neopy_chain_meta_filename, "r").read()
+        if nonce_chain != nonce_container:
+            raise PrivnetWrongChainDatabaseError(
+                "Chain database in Chains/privnet is for a different private network than the current container (or RPC is not enabled). "
```

Comment:
If RPC isn't running, the code should short out above, right? So, I don't think this exception will be raised when RPC isn't running...

Reply:
I think so too. My "RPC is not enabled" comment was also at Line 951, so I think it's just a matter of moving it up there :)

Reply:
@metachris did you get a chance to try out the concern @brianlenz raised? It would be good to make sure the "RPC not enabled" error message gets printed if RPC isn't running.

Reply:
Yeah good points, was a bit tired when adding it here! Fixed with https://github.com/CityOfZion/neo-python/commit/791bd6c017d321e8119c76efe469bfaaf0576038

Reply:
🥇